### PR TITLE
Support pwm

### PR DESCRIPTION
**Pull request by jeffrey870517**
@evshary

### CODE REVIEW

File: examples/c/pwm.c [hunk 1]
```diff
@@ -53,13 +54,12 @@ main(void)
         return EXIT_FAILURE;
     }
 
-#if 0  // ROScube-I doesn't support PWM period setting.
+    // ROScube-I doesn't support PWM period setting.
```
Comment (evshary) [inline examples/c/pwm.c]: Will this affect ROScube-I PWM?

File: src/arm/roscube_pico_nx.c [hunk 2]
```diff
@@ -42,11 +42,306 @@
...
+		{
+			return MRAA_ERROR_NO_RESOURCES;
+		}
+		return MRAA_SUCCESS;
+	}
+	else
+	{
+		return MRAA_ERROR_NO_RESOURCES;
+	}
+	//}
```
Comment (evshary) [inline src/arm/roscube_pico_nx.c]: Don't leave unused code.

File: src/arm/roscube_pico_nx.c [hunk 3]
```diff
@@ -42,11 +42,306 @@
...
+                    return MRAA_ERROR_NO_RESOURCES;
+                }
+            }
+            else
+            {
+                return MRAA_ERROR_NO_RESOURCES;
+            }
+                return MRAA_SUCCESS;
+        }
+        else{
```
Comment (evshary) [inline src/arm/roscube_pico_nx.c]: Suggest following the same coding style in the file.

**Review by evshary — CHANGES_REQUESTED** [review]

File: examples/c/pwm.c (hunk 1 above)
Comment (jeffrey870517) [reply examples/c/pwm.c]: The new ROScube-I doesn't provide PWM ports.

**Pull request merged**